jp/lessons/basics/basics.md

```diff
@@ -6,28 +6,46 @@ order: 1
 lang: jp
 ---
 
-セットアップ、基本型、そして演算。
+入門、基本データ型、そして基本的な演算。
 
 {% include toc.html %}
 
-## セットアップ
+## 入門
 
 ### Elixirのインストール
 
 各OS向けのインストール方法は Elixir-lang.org 上の[Installing Elixir](http://elixir-lang.org/install.html) で探すことができます。
 
+Elixirがインストールされたら簡単にバージョンを確認できます。
+
+    % elixir -v
+    Erlang/OTP 18 [erts-7.3] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
```

Comment:
Sure, @doomspork .
@hinamiyagk, we've just implemented a feature to update the OTP and erts versions easily in further. In order to use it `Erlang/OTP 18 [erts-7.3]` should be replaced with `Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}]`. Examples and description can be found in #441 if needed.